abacusnbody/hod/abacus_hod.py

```diff
@@ -273,7 +273,7 @@ def staging(self):
                     sim_dir / simname / 'halos' / ('z%4.3f' % self.z_mock) / 'halo_info'
                 ).glob('*.asdf')
             )
-        f = asdf.open(halo_info_fns[0], lazy_load=True, copy_arrays=False)
+        f = asdf.open(halo_info_fns[0], lazy_load=True, memmap=False)
```

Comment:
whoops, I messed this one up, it should be `memmap=True`
```suggestion
        f = asdf.open(halo_info_fns[0], lazy_load=True, memmap=True)
```

Reply:
Good catch, although really we don't need to be providing any argument there. We're just using the handle to get info out of the YAML header. I think it should just be `asdf.open(halo_info_fns[0], lazy_load=True)`.

Reply:
I'm not sure if this fits the usage but there is now a way to get just the yaml header https://asdf.readthedocs.io/en/latest/api/asdf.util.load_yaml.html#load-yaml
Although this doesn't do any custom deserialization. As long as the header is only basic YAML types this might work.